.github/CODEOWNERS

```
@@ -4,180 +4,222 @@
+/editor/plugins/script_*.*              @godotengine/script-editor
+/editor/plugins/*shader*.*              @godotengine/shaders
+/editor/themes/                         @godotengine/usability @godotengine/gui-nodes
+/editor/code_editor.*                   @godotengine/script-editor
+/editor/*dock*.*                        @godotengine/docks
+/editor/*shader*.*                      @godotengine/shaders
 
 # Main
 
-/main/                              @godotengine/core
+/main/                                  @godotengine/core
 
 # Misc
 
-/misc/                              @godotengine/buildsystem
-/misc/extension_api_validation/     @godotengine/gdextension @godotengine/dotnet
+/misc/                                  @godotengine/buildsystem
+/misc/extension_api_validation/         @godotengine/gdextension @godotengine/dotnet
 
 # Modules
 
 ## Audio (+ video)
-/modules/minimp3/                   @godotengine/audio
-/modules/ogg/                       @godotengine/audio
-/modules/opus/                      @godotengine/audio
-/modules/theora/                    @godotengine/audio
-/modules/vorbis/                    @godotengine/audio
-/modules/webm/                      @godotengine/audio
+/modules/interactive_music/             @godotengine/audio
+/modules/interactive_music/doc_classes/ @godotengine/audio @godotengine/documentation
+/modules/minimp3/                       @godotengine/audio
+/modules/minimp3/doc_classes/           @godotengine/audio @godotengine/documentation
+/modules/ogg/                           @godotengine/audio
+/modules/ogg/doc_classes/               @godotengine/audio @godotengine/documentation
+/modules/theora/                        @godotengine/audio
+/modules/theora/doc_classes/            @godotengine/audio @godotengine/documentation
+/modules/vorbis/                        @godotengine/audio
+/modules/vorbis/doc_classes/            @godotengine/audio @godotengine/documentation
 
 ## Import
-/modules/basis_universal/           @godotengine/import
-/modules/bmp/                       @godotengine/import
-/modules/cvtt/                      @godotengine/import
-/modules/dds/                       @godotengine/import
-/modules/etc/                       @godotengine/import
-/modules/fbx/                       @godotengine/import
-/modules/gltf/                      @godotengine/import
-/modules/hdr/                       @godotengine/import
-/modules/jpg/                       @godotengine/import
-/modules/pvr/                       @godotengine/import
-/modules/squish/                    @godotengine/import
-/modules/svg/                       @godotengine/import
-/modules/tga/                       @godotengine/import
-/modules/tinyexr/                   @godotengine/import
-/modules/webp/                      @godotengine/import
+/modules/astcenc/                       @godotengine/import
+/modules/basis_universal/               @godotengine/import
+/modules/bmp/                           @godotengine/import
```

```suggestion
/modules/betsy/                         @godotengine/import
/modules/bmp/                           @godotengine/import
```
Will add after CI is done